modules/claims_api/app/controllers/claims_api/v1/forms/power_of_attorney_controller.rb

```diff
@@ -141,6 +141,7 @@ def validate
           validate_poa_code!(poa_code)
           validate_poa_code_for_current_user!(poa_code) if header_request? && !token.client_credentials_token?
           if Flipper.enabled?(:lighthouse_claims_api_poa_dependent_claimants) && form_attributes['claimant'].present?
+            validate_poa_code_by_participant_id!(target_veteran.participant_id, poa_code)
```

Comment:
This check seems faster (cached), so we might as well fail fast if the participant id is bad (before calling BGS in `validate_dependent_by_participant_id!` below) 🤷